sql/core/pom.xml

```diff
@@ -217,6 +217,35 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <configuration>
+          <shadedArtifactAttached>false</shadedArtifactAttached>
+          <artifactSet>
+            <includes>
+              <include>org.apache.parquet:*</include>
+            </includes>
+          </artifactSet>
+          <relocations>
+            <relocation>
+              <pattern>org.apache.parquet</pattern>
+              <shadedPattern>${spark.shade.packageName}.parquet</shadedPattern>
```

Comment:
when did we add `spark.shade.packageName`?

Reply:
https://github.com/apache/spark/blob/82b5ce8c05450674b2922046bed6e14d1a203e6a/pom.xml#L204-L205